alerter.js

```diff
@@ -22,5 +24,6 @@ function alertInCelcius(farenheit) {
 
 alertInCelcius(400.5);
 alertInCelcius(303.6);
+expect(alertFailureCount).greaterThan(0);

```

Comment:
This test says that any number of alerts is ok. But false-positives can be irritating. Strengthen the test to specify the functionality.